twopaneview/examples/windows/DevDay2020KeynoteDemoReactNative/CustomUserControlViewManager.cpp

```diff
@@ -0,0 +1,227 @@
+#include "pch.h"
+#include "CustomUserControlViewManager.h"
+
+#include "JSValueReader.h"
+#include "NativeModules.h"
+
+using namespace winrt;
+using namespace Microsoft::ReactNative;
+using namespace Windows::Foundation;
+using namespace Windows::Foundation::Collections;
+using namespace Windows::Storage::Streams;
+
+using namespace Windows::Devices::Geolocation;
+
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Controls::Maps;
+using namespace Windows::UI::Composition;
+using namespace Windows::UI::Xaml::Hosting;
```

Comment:
Let's not have the complexity of this in the sample - just stick to simple content.

Reply:
This came from the DevDay Demo, so I would not be sure how to remove it without a major layout change.

Reply:
Looks like the sample currently has commented out using the MapControl.  If we want to show something like a map, I suggest faking it with an <Image> component.  Overall, my feeling is we should simplify this example so that it doesn't have any custom viewmanagers.

---
In reply to: [392523181](https://github.com/microsoft/react-native-dualscreen/pull/5#discussion_r392523181) [](ancestors = 392523181)

Reply:
Right, would be very cumbersome to get the MapControl to work, so I removed it for now.